setup.py

```diff
@@ -0,0 +1,25 @@
+import os
+from setuptools import setup, find_packages
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+setup(
+  name="schemas.zcrypto",
+  version="0.0.1",
+  description="ZSchema definitions for zcrypto's JSON output.",
+  classifiers=[
+    "Programming Language :: Python",
+    "Natural Language :: English"
+  ],
+  author="ZMap Team",
+  author_email="zmap-team@umich.edu",
+  url="https://github.com/zmap/zcrypto",
+  keywords="zmap censys ztag internet-wide scanning",
+  packages=["schemas"],
```

Comment:
It might be more natural to have a package name that includes "zcrypto" in it since I could see having multiple ones of these to import and this will be the package that's imported.

Reply:
I'll see if I can find an example of this (e.g. multiple different components dropping modules under `http.*`); right now, both `zgrab2` and `zcrypto` exist under `schemas` and it seems to "work", but that was just an empirical finding.